src/lib/IDE/Intellij.py

```diff
@@ -0,0 +1,170 @@
+import logging
+import time
+
+from src.lib.application import Application
+from src.lib.config import config_data
+
+
+class Intellij(Application):
+    """
+    Class for managing IntelliJ application
+    """
+
+    def close_ide(self):
+        """
+        Closes the IDE
+        """
+        self.click_element(locator_type="image", locator="file_menu.png")
+        self.press_keys("up")
+        self.press_keys("enter")
+        time.sleep(1)
+        self.press_keys("enter")
+        # try:
```

Comment:
@midays , could you review these lines ? It looks like an oversight to me.

Reply:
When exiting the IDE there was issues with the exit button, for some reason the framework fails to locate it ,so I replaced it with "enter" click, and i added 1 second delay to wait for the popup to completely load

Reply:
@midays How is it exiting the IDE . I do not see any click on exit button .

Reply:
it click on the file button in the menu> move the arrow up ( which will move the highlight to the lat button i.e. exit button ), then click enter, then wait for 1 sec, then click enter again which clicks the exit button in the dialog